app/views/layouts/login.html.erb

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Kata07RelojChecadorK7F22T4</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag "sass/pages/log_in", "data-turbo-track": "reload" %>
+    <%= javascript_importmap_tags %>
+  </head>
+
+  <body>
+    <div class="container">
+      <%= yield %>
+    </div>
+  </body>
+</html>
```

Comment:
The application.html.erb file is not your main view?

The question is oriented in the sense that I observe that you have body and head tags that only correspond to that file.

Reply:
We are using SASS for our styles, and when we used application.html.erb some styles were overwritten. So we try to create a specific layout for some views and specify the CSS file for this layout.

But we are not sure if this is the best way or the best practice to solve this problem.

Reply:
@Frcomparan, You can assign a sass file to each of the views individually, which can give precedence to the style of application.html.erb

It is not recommended to put the head or body tags to another file, since the application.html.erb is by default the base of the project, but still each view can use its own sass and js files.

Reply:
@YilKanda I've been looking for information to implement your advice. I find a solution, implemented it and worked fine. Thanks